pkg/controller/import-controller.go

```diff
@@ -748,6 +753,19 @@ func getEndpoint(pvc *corev1.PersistentVolumeClaim) (string, error) {
 	return ep, nil
 }
 
+// returns the import image part of the endpoint string
+func getImportImage(pvc *corev1.PersistentVolumeClaim) (string, error) {
+	ep, err := getEndpoint(pvc)
+	if err != nil {
+		return "", nil
+	}
+	parts := strings.Split(ep, "://")
```

Comment:
url.Parse uses existing libraries to do the parsing.

Reply:
Fixed